src/workerd/api/kv.h

```diff
@@ -48,11 +48,13 @@ class KvNamespace: public jsg::Object {
   struct GetWithMetadataResult {
     GetResult value;
     kj::Maybe<jsg::Value> metadata;
+    kj::Maybe<jsg::Value> cacheStatus;
```

Comment:
@jasnell can't `jsg::Value` represent null? is there a reason to return `kj::Maybe<jsg::Value>` instead of `jsg::Value`?